sklearn/gaussian_process/tests/test_kernels.py

```diff
@@ -46,6 +46,10 @@
     if metric in ["additive_chi2", "chi2"]:
         continue
     kernels.append(PairwiseKernel(gamma=1.0, metric=metric))
+kernels += [SelectDimensionKernel(RBF(length_scale=2.0),
+                                  active_dim=np.array([0])),
+            SelectDimensionKernel(RBF(length_scale=2.0),
+                                  active_dim=np.array([False, True]))]
```

Comment:
could you also add a kernel where more than one dimensions are active?  what would happen if no dimension is active